README.md

```diff
@@ -4,6 +4,9 @@
 
 GitHub Action for creating a GitHub App installation access token.
 
+Thanks to the provided [typings](action-types.yml), it is possible to use this action in a type-safe way using
```

Comment:
This reads to me as an endorsement or advertisement for a 3rd party tool to write workflows.

Actions cannot validate nor guarantee the security or safety of this tool, so I do not think we should mention it so prominently in the README.